zerver/tests/test_home.py

```diff
@@ -221,11 +221,11 @@ def _sanity_check(self, result):
 
     def test_terms_of_service(self):
         # type: () -> None
-        email = 'hamlet@zulip.com'
+        user = self.example_user('hamlet')
+        email = user.email
         self.login(email)
 
         for user_tos_version in [None, '1.1', '2.0.3.4']:
-            user = get_user_profile_by_email(email)
             user.tos_version = user_tos_version
             user.save()
```

Comment:
Yeah, these changes were mostly automated, and I tried to mostly protect against this sort of thing by having my scripts be conservative in what they would change.  Hopefully this is the only one that slipped through.  Obviously, some mis-re-factorings would just break, but the cases of reloading the UserProfile can be kind of subtle.  The remaining stuff is gonna be done more manually.